README.md

```diff
@@ -27,6 +27,52 @@ expected syntax (and semantics) for all such values is that of
 [Perl regular expressions](https://perldoc.perl.org/perlre.html#Regular-Expressions).
 
 
+### Accounts
+
+ledger2beancount will convert ledger account declaration to beancount
+`open` statements using the `account_open_date` variable as the opening
+date.  The `note` is used as the `description`.
+
+You can also add account mappings to `account_map` for automatic
+conversion of account names.
+
+
+### Commodities
+
+Like accounts, ledger2beancount will convert ledger commodity
+declarations to beancount.  The `note` is converted to `name`.
+
+ledger2beancount will automatically convert all commodities to uppercase
+and strip quoted commodities.  If you require a mapping between ledger
+and beancount commodities, you can use `commodity_map`.
+
+Commodity symbols (like $, € and £) are not supported.
+
+
+### Flags
+
+ledger2beancount supports both transaction flags and account flags.
+
+
+### Dates
+
+Only dates in the format YYYY-MM-DD (ISO 8601) are currently supported.
+
+
+### Auxiliary dates
+
+Beancount currently doesn't support auxiliary dates so these are stored
```

Comment:
let's link to https://www.ledger-cli.org/3.0/doc/ledger3.html#Auxiliary-dates here